unattended_scripts/wazuh_install.sh

```diff
@@ -286,22 +282,24 @@ main() {
         importFunction "kibana.sh"
 
         if [ -n "${ignore}" ]; then
-            logger -w "Health-check ignored."
+            logger -w "Health-check ignored for AIO"
         else
             healthCheck AIO
         fi
-        checkSystem
-        installPrerequisites
-        addWazuhrepo
         installWazuh
         installElasticsearch
         configureElasticsearchAIO
         installFilebeat
         configureFilebeatAIO
         installKibana
         configureKibanaAIO
+    fi
+
+    if [ -n "${AIO}" ] || [ -n "${wazuh}" ] || [ -n "${kibana}" ] || [ -n "${elastic}" ]; then
+        logger -w "restoreWazuhrepo"
```

Comment:
Not a very explanatory comment. A better option would be "Restoring Wazuh repository"

Reply:
Done